docs/victory-zoom-container/docs.md

```diff
@@ -0,0 +1,130 @@
+# VictoryZoomContainer
+
+`VictoryZoomContainer` provides pan and zoom behavior for any Victory component that works with an
+x, y axis. To add panning and zooming behavior, add `VictoryZoomContainer` as the
```

Comment:
I'm not sure on this one, but i think the noun might be `x-y axis`, not `x, y axis`.